django_irods/views.py

```diff
@@ -73,7 +70,7 @@ def download(request, path, use_async=True,
     is_aggregation_request = request.GET.get('aggregation', "False").lower() == "true"
     api_request = request.META.get('CSRF_COOKIE', None) is None
     aggregation_name = None
-    is_sf_request = False
+    is_sf_request = True
```

Comment:
This variable seems to be always set to True

Reply:
whoops, thanks for catching that. I believe I could just flip the initialization back to False. I think this was updated because I made an attempt to refactor some of this that I retreated from.